lib/niso/jats/alt_title.rb

```diff
@@ -48,7 +11,6 @@ class AltTitle < Lutaml::Model::Serializable
       attribute :id, :string
       attribute :specific_use, :string
       attribute :lang, :string
-      include BasicText
```

Comment:
Why did you drop it here and another places? Could this break some functionality of the gem?